docs/current_docs/api/fs-filters.mdx

```diff
@@ -85,6 +85,14 @@ Here's an example of a Dagger Function that excludes everything in a given direc
 ```typescript file=./snippets/fs-filters/pre-call/typescript/index.ts
 ```
 
+</TabItem>
+<TabItem value="PHP">
+
+Here's an example of a Dagger Function that excludes everything in a given directory except TypeScript source code files:
```

Comment:
```suggestion
Here's an example of a Dagger Function that excludes everything in a given directory except PHP source code files:
```

Reply:
Thank you! [Resolved in this commit](https://github.com/dagger/dagger/pull/9917/commits/35d43d8c772c99c3d7912057dc7fc3a8a8e3816c) :+1: